R/chromote.R

```diff
@@ -302,6 +302,13 @@ Chromote <- R6Class(
         private$sessions[[sid]] <- NULL
         session$mark_closed()
       })
+
+      # When a target crashes, raise a warning.
+      # The session cannot be closed because no session id is returned by the
+      # Inspector.targetCrashed event
+      self$protocol$Inspector$targetCrashed(function(msg) {
+        warning("A target has crashed.")
```

Comment:
How about:

```suggestion
        warning("Chromote has received a Inspector.targetCrashed event. This means that the Chrome/Chromium process has probably crashed.")
```

Is this message accurate? I don't really know anything about the `Inspector.targetCrashed` event.

Reply:
```suggestion
        warning("Chromote has received a Inspector.targetCrashed event. This means that the ChromoteSession has probably crashed.")
```
When `Inspector.targetCrashed` fires, the Chrome process still runs, only the target/tab has crashed.

I have got the `Inspector.targetCrashed` event only in docker. In this context, the session has been lost.
Lauching Chrome with the `--disable-dev-shm-usage` flag has always solved the problem, that's why `pagedown::chrome_print()` gives this hint: https://github.com/rstudio/pagedown/blob/f9779e38a4b9652b111fc763c414ea61998cb208/R/chrome.R#L637